opcua/server/binary_server_asyncio.py

```diff
@@ -113,7 +113,7 @@ def _process_data(self, data):
             sockname = self._server.sockets[0].getsockname()
             self.hostname = sockname[0]
             self.port = sockname[1]
-        print('Listening on {0}:{1}'.format(self.hostname, self.port))
+        self.logger.info('Listening on {0}:{1}'.format(self.hostname, self.port))
```

Comment:
Not sure what should be done here. I had it as a warning earlier. That is the kind of things one want to see when running it main thread but does not want when running in another thread or submodule...

Reply:
I’d have assumed that, especially when not trying around with one of the example-code-pieces, you’d configure your logger to your liking. What could make sense is having it as a logger.warning, so it’s the only higher-severity line emitted by the BinaryServer, and users can more easily write down what they wanna see — also for example in the commandline server or examples.

How does that sound to you?

Reply:
ok make it a warning again.